docs/tutorials/11_quadratic_hamiltonian_and_slater_determinants.ipynb

```diff
@@ -0,0 +1,440 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "a2ecf1af-4d9d-482c-8cda-44ae649e40c3",
+   "metadata": {},
+   "source": [
+    "# Quadratic Hamiltonians and Slater determinants\n",
+    "\n",
+    "A quadratic Hamiltonian is a Hamiltonian of the form\n",
+    "$$\n",
+    "    H =\n",
+    "    \\sum_{j, k} M_{jk} a^\\dagger_j a_k\n",
+    "    + \\frac12 \\sum_{j, k}\n",
+    "        (\\Delta_{jk} a^\\dagger_j a^\\dagger_k + \\Delta^*_{jk} a_k a_j)\n",
+    "    + \\text{constant}\n",
+    "$$\n",
+    "where $M$ is a Hermitian matrix ($M^\\dagger = M$) and $\\Delta$ is an antisymmetric matrix ($\\Delta^T = -\\Delta$), and the $\\{a_j\\}$ are fermionic annihilation operators which satisfy the anticommutation relations\n",
```

Comment:
```
and the $\\{a_j\\}$ are fermionic annihilation operators
```
I suggest to be a bit more explicit here and also include the creation operators (the same way you do it in the documentation).

Reply:
Done.